### DISCUSSION TITLE
在vite工程中，node_modules下的antd/dist下面没有reset.css

### DISCUSSION THREAD

**Discussion by Sunny-117** [ant-design/ant-design#40012] [Q&A]
改为 import 'antd/dist/antd.css'; 生效

**Comment by Sunny-117** [comment]
<img width="425" alt="image" src="https://user-images.githubusercontent.com/73089592/210565091-a43d64d3-0f87-43e9-921c-a91227482d36.png">

**Reply by yoyo837** [reply]
你这个包不是5.x吧
